code/src/main/java/com/google/sps/data/Business.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+
+@Entity
+public class Business {// This class is solely to check if a business was contacted in the last week.
```

Comment:
Please use Javadoc (here and throughout) on public methods and classes.

Javadoc best practices: https://www.tutorialspoint.com/java/java_documentation.htm

Reply:
Most of this CL will be comments.

Reply:
Just re-upping this one last time. Every public class, and most public methods should have a javadoc describing what it does/is. 

Also throughout this change, we should make fields private, and have getters/setters. Getters/setters typically don't require documentation, unless they have additional logic.

Reply:
Does every variable need getters and setters? There already are getters and setters for some variables.

Reply:
I thought about doing Javadoc but wasn't sure if it goes before or after @Entity. Also should I be adding @author and so on?

Reply:
No need to add author. The important part is doing the description of _what it is_ alongside a description of any _no obvious or nonintuitive things about it_. Describing things in english is far easier for a person who isn't familiar with the code or ideas behind it to grok.

Your goal with documentation is to make it so that a technically capable reader without knowledge of project could navigate it and make changes effectively. Public interfaces are a huge part of that goal, so each should have a description of what they do.

Yes - we should not have publicly visible fields, so we will need getters on each. We might not need setters on each if we expect them to be readonly.

Reply:
I added comments.

Reply:
I added comments and made everything private with getters.